toxicity/feature_adder.py

```diff
@@ -27,8 +28,30 @@ def __init__(self, data_dir="data", upper_case=False, word_count=False, unique_w
             self._count_little_case: little_case,
             self._count_stopwords: stopwords,
             self._question_or_exclamation: question_or_exclamation,
-            self._count_bad_words: number_bad_words
-        }
+            self._count_bad_words: number_bad_words,
+            self._polarity_subjectivity_score: sentiment_analysis
+            }
+
+    def _polarity_subjectivity_score(self, df):
```

Comment:
It appears that this function adds 2 features. Should we break it into 2 functions in case the user only wants 1 of the 2 features for example? Or do they only make sense together (in which case we should keep it like this)